### PR TITLE
Adding docs for the Lumi Keys Studio Edition integration

### PR DESCRIPTION
**Pull request by litui**
As described.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ In addition, a number of improvements have been made to how the OLED display is
 - Fixed crash when using an external controller to control ModFX.
 - Fixed CC74 on the MPE master channel behaving like an expression event.
 
+### 3rd Party Device Integration
+
+- Hook support for adding signals to third-party devices starting with support for LED key highlighting and colours on the Lumi Keys Studio Edition.
+
```
Comment (seangoodvibes) [inline CHANGELOG.md:76]: Under User Interface above we have a point on the LUMI Keys.

"- Added support for LUMI Keys SYSEX protocol protocol. When hosting a LUMI Keys keyboard, the current scale will automatically be set on the keyboard."

I like the 3rd Party Device Integration section though. I would suggest removing the bullet from the User Interface section and maybe combine the wording from both. 

E.g. "Added support for third-party devices SYSEX protocol integration starting with the LUMI Keys Studio Edition. When hosting a LUMI Keys keyboard, the current scale and key colours will be automatically set."

**Review by seangoodvibes — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (litui) [reply CHANGELOG.md:76]: Ah alright. Missed that. I'll remove my CHANGELOG.md addition.

**Review by seangoodvibes — APPROVED** [review]

**Review by seangoodvibes — CHANGES_REQUESTED** [review]
Can you add link to your PR at the bottom of the community features doc? Thx! It will make your [#812] reference work :)

**Comment by litui** [comment]
Aye aye

**Comment by litui** [comment]
Added!

**Review by seangoodvibes — APPROVED** [review]

**Comment by seangoodvibes** [comment]
LGTM! Thanks :D

**Comment by soymonitus** [comment]
@litui Have you tested the new scales added to Deluge with the Lumi? I modified the code of the Lumi scale detector to search for all the transposition of a scale, so it might be buggy as I didn't have a Lumi to test

**Pull request merged**